board/views.py

```diff
@@ -100,25 +100,57 @@ class HomePage(TemplateView):
     """
     template_name = 'home.html'
 
+LOG_DIR = "/Users/kay/recboard/model_summaries"
 
 def create(request):
     print ('Parent process pid:', os.getpid())
-    PATH_TO_DATASET = '/ctrsr_datasets/citeulike-a/users.dat'
+    PATH_TO_DATASET = '/files/data/users.dat'
     model_controller_obj = ModelManager('BPR', 'train_samp', 'val_samp', 'test_samp', 'AUC', PATH_TO_DATASET)
-
+    
     p = Process(target=ModelManager.sample_data_and_train, args=(model_controller_obj,))
     p.start()
     p.join(2)
 
-    try:
-        generated_model_id = model_controller_obj.model_id
-        err = "No err "
-        return HttpResponse(str(generated_model_id + err))
-    except:
-        print("Error in getting generated model-id")
-        p.terminate() # Force terminate training
-        return HttpResponse("Error in getting generated model-id")
+    # try:
+    generated_model_id = model_controller_obj.model_id
+    err = "No err "
+    user = get_dummy_user()
+    if not user.datasets: 
+        return JsonResponse({'msg':'No datasets found'})
+
+    if not user.workspaces:
+        user.workspaces.append(Workspace(name="workspacename2"))
+        db.insert('user',user)
+    
+    # TODO: maintain a list of available ports later
```

Comment:
i started an issue for this. feel free to start issues for your TODO's. considering we are in hurry rn.

Reply:
👍